Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -27,11 +28,11 @@ public sealed class TemperatureSystem : EntitySystem
     ///     This is done because both AtmosExposed and Flammable call ChangeHeat in the same tick, meaning
     ///     that we need some mechanism to ensure it doesn't double dip on damage for both calls.
     /// </summary>
-    public HashSet<TemperatureComponent> ShouldUpdateDamage = new();
+    public HashSet<EntityUid> ShouldUpdateDamage = new();
 
     public float UpdateInterval = 1.0f;
 
-    private float _accumulatedFrametime;
+    private float _accumulatedFrameTime;
```

Comment:
There are still a lot of other unrelated changes like this to the content of your PR. We can still review with changes like this, but it makes it more difficult and as a result PRs can get buried like this one.

The PR guidelines are designed to make PRs easy to review so that actual content changes are clearly separate from no-op changes like this and so that PRs get in faster. Your help with this is appreciated.

Reply:
Ok I see, thank you for clarifying. I did not actually plan any of those changes going in, the thought process was that if I am touching this system I should just as well clean up every thing Rider was warning and nagging about, so the next person gets to work with a pristine file with a more easily readable overview. 
I figured that putting them in a distinct commit before the important changes will be enough, but I'll move them to their own PR then